src/packcc.c

```diff
@@ -3360,6 +3365,10 @@ static bool_t generate(context_t *ctx) {
             "#define PCC_FREE(auxil, ptr) free(ptr)\n"
             "#endif /* !PCC_FREE */\n"
             "\n"
+            "#ifndef PCC_DEBUG\n"
+            "#define PCC_DEBUG(event, rule, level, pos, buffer, length)\n"
```

Comment:
It's better to define the empty macro like this.

    #define PCC_DEBUG(event, rule, level, pos, buffer, length) ((void)0)